l10n_it_fiscalcode/__openerp__.py

```diff
@@ -20,7 +20,6 @@
 ##############################################################################
 {
     'name': 'Italian Localisation - Fiscal Code',
-    'version': '8.0.0.1.0',
```

Comment:
Tutti a rifa' sto cidice_fiscale :D cmq qui trovi il lavoro che a suo tempo avevo fatto per la v9 ed il porting alla v10:

https://github.com/gigidn/l10n-italy/tree/v10-fiscalcode

Il campo individual serve perchè le ditte individuali hanno codice fiscale = codice_fiscale_proprietario e non alla partita iva.

Il codice fiscale delle amministrazioni pubbliche è composto da soli numeri ma non è una partita iva valida se applichi le regole di verifica della partita iva.

Come dissi tempo fa a me non piace un modulo codice_fiscale con calcolo integrato ... meglio dividere in due il modulo, uno con la sola funzione del codice fiscale e l'altro che lo estende ed aggiunge la creazione automatica cosi' che se uno vuole evita di installare questo ultimo perchè fonte di mille problemi.

In ogni modo se puo' essere da spunto i codici son li .... che poi son gli stessi o quasi della PR di molti mesi fa.